### PR TITLE
Normalize imports: numpy as np, jax.numpy as jnp

### PR DESCRIPTION
**Pull request by maxentile**
Small cleanup: Applies convention suggested by @mcwitt and discussed last week:
* original numpy should always be `np`
* jax.numpy should always be  `jnp`

-----

Notes:
* This DOES update references within code but does NOT update within strings or comments
* This does NOT change the couple places where `jax.numpy.eye` or `jax.numpy.sum` were used
* This should later be added to .git-blame-ignore-revs
* To scan for any unwanted changes, reviewing commit by commit may be easiest
* This does not add a linting stage to enforce this convention, but that may be something to consider!

### CODE REVIEW

File: tests/test_jax_nonbonded.py [hunk 1]
```diff
@@ -47,16 +47,16 @@
 def resolve_clashes(x0, box0, min_dist=0.1):
     def urt(x, box):
         distance_matrix = distance(x, box)
-        i, j = np.triu_indices(len(distance_matrix), k=1)
+        i, j = jnp.triu_indices(len(distance_matrix), k=1)
```
Comment (proteneer) [inline tests/test_jax_nonbonded.py:50]: not important for merging: but i'm surprised that these used jnp!

File: tests/test_jax_nonbonded.py [hunk 2]
```diff
@@ -122,10 +122,10 @@ def generate_waterbox_nb_args() -> NonbondedArgs:
     cutoff = nb.get_cutoff()
 
     lamb = 0.0
-    charge_rescale_mask = onp.ones((N, N))
-    lj_rescale_mask = onp.ones((N, N))
-    lambda_plane_idxs = np.zeros(N, dtype=int)
-    lambda_offset_idxs = np.zeros(N, dtype=int)
+    charge_rescale_mask = np.ones((N, N))
+    lj_rescale_mask = np.ones((N, N))
+    lambda_plane_idxs = jnp.zeros(N, dtype=int)
```
Comment (proteneer) [inline tests/test_jax_nonbonded.py:127]: should probably fix this later as well!

File: tests/test_jax_utils.py [hunk 3]
```diff
@@ -30,34 +30,34 @@ def test_delta_r():
...
 
     for _ in range(5):
-        n_atoms = onp.random.randint(50, 1000)
-        dim = onp.random.randint(3, 5)
-        ri, rj = onp.random.randn(2, n_atoms, dim)
-        box = np.eye(dim)
+        n_atoms = np.random.randint(50, 1000)
+        dim = np.random.randint(3, 5)
+        ri, rj = np.random.randn(2, n_atoms, dim)
+        box = jnp.eye(dim)
```
Comment (proteneer) [inline tests/test_jax_utils.py:39]: should probably be `np`

File: tests/test_jax_utils.py [hunk 4]
```diff
@@ -119,13 +119,13 @@ def test_batched_neighbor_inds():
     """
     n_confs, n_particles, dim = 100, 1000, 3
 
-    confs = onp.random.rand(n_confs, n_particles, dim)
+    confs = np.random.rand(n_confs, n_particles, dim)
     cutoff = 0.3
 
-    boxes = np.array([np.eye(3)] * n_confs)
+    boxes = jnp.array([jnp.eye(3)] * n_confs)
```
Comment (proteneer) [inline tests/test_jax_utils.py:125]: should probably be `np`

File: tests/test_jax_utils.py [hunk 5]
```diff
@@ -119,13 +119,13 @@ def test_batched_neighbor_inds():
...
 
     n_alchemical = 50
-    pairs = pairs_from_interaction_groups(np.arange(n_alchemical), np.arange(n_alchemical, n_particles))
+    pairs = pairs_from_interaction_groups(jnp.arange(n_alchemical), jnp.arange(n_alchemical, n_particles))
```
Comment (proteneer) [inline tests/test_jax_utils.py:128]: should probably be `np`

File: tests/test_jax_utils.py [hunk 6]
```diff
@@ -142,4 +142,4 @@ def d(conf, pairs, box):
     neighbor_distances = vmap(d)(confs, batch_pairs, boxes)
 
     assert neighbor_distances.shape == (n_confs, n_neighbor_pairs)
-    assert np.sum(neighbor_distances < cutoff) == np.sum(full_distances < cutoff)
+    assert jnp.sum(neighbor_distances < cutoff) == jnp.sum(full_distances < cutoff)
```
Comment (proteneer) [inline tests/test_jax_utils.py:145]: should probably be `np`

File: timemachine/optimize/protocol.py [hunk 7]
```diff
@@ -115,7 +115,7 @@ def log_weights_from_mixture(u_kn: Array, f_k: Array, N_k: Array) -> Array:
     mixture of states p(x) = (1 / K) \sum_k e^-u_k / Z_k
     """
     log_q_k = f_k - u_kn.T
-    N_k = np.array(N_k, dtype=np.float64)  # may be ints, or in a list...
+    N_k = jnp.array(N_k, dtype=jnp.float64)  # may be ints, or in a list...
```
Comment (proteneer) [inline timemachine/optimize/protocol.py:118]: should probably be `np`

File: timemachine/potentials/evp.py [hunk 8]
```diff
@@ -191,45 +191,45 @@ def test1():
...
-        a_com, a_tensor = inertia_tensor(x_a, onp.ones(N, dtype=np.float64))
+        a_com, a_tensor = inertia_tensor(x_a, np.ones(N, dtype=jnp.float64))
 
-        onp_res = onp.linalg.eigh(a_tensor)
+        onp_res = np.linalg.eigh(a_tensor)
         w = onp_res[0]
         Q = onp_res[1]
         for d in range(3):
-            onp.testing.assert_almost_equal(np.matmul(a_tensor, Q[:, d]), w[d] * Q[:, d])
+            np.testing.assert_almost_equal(jnp.matmul(a_tensor, Q[:, d]), w[d] * Q[:, d])
```
Comment (proteneer) [inline timemachine/potentials/evp.py:221]: should probably be `np`

File: timemachine/potentials/evp.py [hunk 9]
```diff
@@ -191,45 +191,45 @@ def test1():
...
 
-        jnp_res = np.linalg.eigh(a_tensor)
+        jnp_res = jnp.linalg.eigh(a_tensor)
         evp_res = dsyevv3(a_tensor)
 
-        np.set_printoptions(formatter={"float": lambda x: "{0:0.16f}".format(x)})
+        jnp.set_printoptions(formatter={"float": lambda x: "{0:0.16f}".format(x)})
```
Comment (proteneer) [inline timemachine/potentials/evp.py:226]: should probably be `np`

File: timemachine/potentials/nonbonded.py [hunk 10]
```diff
@@ -277,11 +277,11 @@ def nonbonded_v3_on_specific_pairs(conf, params, box, pairs, beta: float, cutoff
     # distances and cutoff
     dij = distance_on_pairs(conf[inds_l], conf[inds_r], box)
     if cutoff is None:
-        cutoff = np.inf
+        cutoff = jnp.inf
```
Comment (proteneer) [inline timemachine/potentials/nonbonded.py:280]: should probably be `np`

File: timemachine/potentials/shape.py [hunk 11]
```diff
@@ -106,7 +106,7 @@ def harmonic_overlap(conf, params, box, lamb, a_idxs, b_idxs, alphas, weights, k
     conf_a = conf[a_idxs]
     conf_b = conf[b_idxs]
 
-    params_c = np.stack([alphas, weights], axis=1)
+    params_c = jnp.stack([alphas, weights], axis=1)
```
Comment (proteneer) [inline timemachine/potentials/shape.py:109]: should probably be `np`

**Review by proteneer — APPROVED** [review]
lgtm! just documenting some places that we'd want to manually change `jnp` -> `np`

File: tests/test_jax_nonbonded.py (hunk 1 above)
Comment (maxentile) [reply tests/test_jax_nonbonded.py:50]: Agreed!

Will prefer to address this in a separate PR so this one can be purely mechanical / added to `.git-blame-ignore-revs`

**Review by mcwitt — APPROVED** [review]
LGTM! Double-checked and don't find any instances of the old import aliases (except one in commented-out code)

**Comment by maxentile** [comment]
Thanks! Will get this merged, then address the requested list of manual changes in a separate PR
